File: 5KYU/Human Readable Time/solution.cs
using System;
public static class TimeFormat
{
    public static string GetReadableTime(int seconds)
    {
		if (seconds == 0)
			return "00:00:00";

		var hours = seconds / 3600;
		seconds = seconds - (hours * 3600);
		var minutes = 0;
		if (seconds != 0)
		{
			minutes = seconds / 60;
			seconds = seconds - (minutes * 60);
		}
		return $"{hours:00}:{minutes:00}:{seconds:00}";
    }
}

File: 5KYU/Human Readable Time/solution.go
package kata

import "fmt"

func HumanReadableTime(seconds int) string {
	hours := seconds / 3600
	seconds -= hours * 3600
	minutes := seconds / 60
	seconds -= minutes * 60

	return fmt.Sprintf("%02d:%02d:%02d", hours, minutes, seconds)
}
